techniques/android/MASTG-TECH-0131.md

```diff
@@ -5,20 +5,24 @@ platform: android
 
 Although other tools like Maven exist, we will focus on [Gradle](https://developer.android.com/build/dependencies), the default build automation and dependency management tool in Android Studio.
 
-Tools like @MASTG-TOOL-0131 can be integrated into build environments where they can inspect build configuration files, actual .jar files, and their metadata (e.g., version) to identify dependencies. These tools then compare the dependencies to a vulnerability database, such as the National Vulnerability Database (NVD), to identify known vulnerabilities.
+Tools like @MASTG-TOOL-0131 can be integrated into build environments where they can inspect build configuration files, actual `.jar` files, and their metadata (e.g., version) to identify dependencies. These tools then compare the dependencies to a vulnerability database, such as the National Vulnerability Database (NVD), to identify known vulnerabilities.
 
 To test for dependencies with known vulnerabilities, integrate the @MASTG-TOOL-0131 plugin into the Android project via Gradle. The Android project's dependencies are located in the directory `~/.gradle/caches/modules-2/files-2.1` and not in the Android project directory.
 
 The dependencies are integrated into the project at runtime but are also modified and compiled into the DEX file(s) of the APK; therefore, we must scan the dependencies in Android Studio, not the APK.
 
 Before running the scan, obtain an API key for NVD to retrieve the latest CVE information. You can request the API key to access the NVD API from <https://nvd.nist.gov/developers/request-an-api-key>.
 
+!!!info
+
+    In recent versions of @MASTG-TOOL-0131 (up to and including 12.1.1 at the time of writing), you may encounter multiple 'NoSuchMethodError' messages relating to 'ZipFile.builder()'. This can be mitigated by [pinning the  version of `org.apache.commons:commons-compress`](https://github.com/dependency-check/DependencyCheck/issues/7405#issuecomment-2785588330).
```

Comment:
```suggestion
!!! info

    In recent versions of @MASTG-TOOL-0131 (up to and including version 12.1.1 at the time of writing), you may encounter multiple 'NoSuchMethodError' messages related to `ZipFile.builder()`. This can be resolved by [pinning the version of `org.apache.commons:commons-compress`](https://github.com/dependency-check/DependencyCheck/issues/7405#issuecomment-2785588330).
```